component/site/libraries/helper.php

```diff
@@ -509,7 +509,7 @@ function SetMetaTags()
 	}
 
 	public static
-			function forceIntegerArray(&$cid, $asString = true)
```

Comment:
We can't just change this without changing some places where its used assuming the array is passed by reference e.g. in dbmodel accessiblecategorylist and icals controller :(